src/components/layout/DefaultLayout.tsx

```diff
@@ -3,15 +3,20 @@ import { Content, Flex } from '@adobe/react-spectrum';
 
 import PublicHeader from '../PublicHeader';
 import { useUser } from '../../hooks';
+import { ZetkinOrganization } from '../../interfaces/ZetkinOrganization';
 
-const DefaultLayout : FunctionComponent = ({ children }) => {
+interface DefaultLayoutProps {
+    org?: ZetkinOrganization;
+}
+
+const DefaultLayout : FunctionComponent<DefaultLayoutProps> = ({ children, org }) => {
     const user = useUser();
 
     return (
         <Flex
             direction="column"
             gap="size-100">
-            <PublicHeader user={ user }/>
+            <PublicHeader org={ org || null } user={ user }/>
```

Comment:
The fallback to `null` should not be necessary. Just passing `org` as it is should be enough, since the `PublicHeader` component knows how to handle `undefined` now, doesn't it?

Reply:
Yes, that works!